packages/react/src/components/Storage/FileUploader/UploadTracker/index.tsx

```diff
@@ -90,7 +90,7 @@ export function UploadTracker({
               size="small"
               onClick={() => {
                 setTempName(name);
-                onCancelEdit();
+                onCancelEdit?.();
```

Comment:
shouldn't we check if it's a function first, rather than just that it's non-nullish?

Reply:
**onCancelEdit** is defined as ``onCancelEdit?: () => void;`` so only checked to make sure it is not undefined before calling it.